mock/mock.go

```diff
@@ -191,6 +191,10 @@ type Mock struct {
 	// Holds the calls that were made to this mocked object.
 	Calls []Call
 
+	// T is An optional variable that holds the test struct, to be used when an
+	// invalid mock call was made.
+	T TestingT
```

Comment:
Can you unexport `TestingT`?

The package already exports too much of it's internals and we should keep new things to a minimum.

Reply:
I didn't add the `TestingT` interface you mean unexport `T` I guess?